.github/workflows/update-browser-pin.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will update the version in .browser and open a PR
+
+name: 'Update pinned Chrome'
+
+on:
+  schedule:
+    # Run hourly at xx:05.
+    - cron: '05 * * * *'
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: npm
+      - uses: google/wireit@setup-github-actions-caching/v1
+      - name: Install and build npm dependencies
+        run: npm ci
+      - name: Download the latest version
+        run: |
+          browser_spec="$(npx @puppeteer/browsers install chrome@canary --path /tmp/browsers | cut -f 1 -d' ')"
+          echo "$browser_spec\n" > .browser
```

Comment:
`echo` emits `\n` by default. This will yield:

```
1 chrome@123
2
```

I believe you want only

```
1 chrome@123
```

...in which case, omit the `\n`.